src/main.cpp

```diff
@@ -351,6 +353,8 @@ int main(int argc, char **argv) {
             world.updateGameObjects(deltaTime, totalTime);
             camera.update(deltaTime);
 
+            gameManager.decreaseTime(deltaTime);

```

Comment:
We'll need to do some check in this loop to determine if the time has run out. If so we should stop the game.